en/docs/learn/consume-api/invoke-apis/invoke-apis-using-tools/try-out-using-postman.md

```diff
@@ -1,57 +1,56 @@
-# Try out using Postman.
+# Try out using Postman
 
-Let`s see how to download a postman collection.
+Let`s see how to download a Postman collection.
 
-1.  Sign in to the WSO2 Developer Portal (`https://<hostname>:9443/devportal`) and click an API (e.g. `Pizza` ) to go to the API overview.
+1.  Sign in to the WSO2 Developer Portal (`https://<hostname>:9443/devportal`) and click an API (e.g., `Pizza` ) to go to the API overview.
 
-    ![({{base_path}}/assets/attachments/postman/postman_api_overview.png)]({{base_path}}/assets/attachments/postman/postman_api_overview.png)
```

Comment:
Just changing the path of the image in the doc is not enough. You need to change the location of the physical image of the location as well. So move all the images that u added newly to the `en/docs/assets/img/learn/` folder.

In addition, please delete the folder named "postman" that u added in the "en/docs/assets/attachments/" folder.

Reply:
Done with the final commit Akka.